elliottlib/errata.py

```diff
@@ -24,6 +26,7 @@
 from errata_tool import Erratum, ErrataException, ErrataConnector
 
 ErrataConnector._url = constants.errata_url
+ErrataXMLRPC = xmlrpclib.ServerProxy(constants.errata_xmlrpc_url)
```

Comment:
A few issues:

1. I think it is better to use an underscore name.
2. Just found `xmlrpclib` is renamed to `xmlrpc.client` in Python 3. But anyway we don't have to worry about that in this PR.

BTW I tried your PR and found the XMLRPC method you called doesn't require Kerberos authentication!

Reply:
done